### ISSUE TITLE
Does not work with Warcraft 3 (fullscreen)

### ISSUE THREAD

**Issue by bloor** [lmachens/skeleton#22]
Does not work with Warcraft 3 (fullscreen / DX9)

**Comment by lmachens** [comment]
Yes, the window need's to be borderless. This could be an issue for old games.
You could try out tools like https://github.com/Codeusa/Borderless-Gaming to play WC3 in borderless mode.
